docs/install/from_source.rst

```diff
@@ -205,7 +205,7 @@ like ``virtualenv``.
 
    .. code:: bash
 
-       pip3 install --user numpy decorator attrs
+       pip3 install --user numpy decorator attrs typed_ast pytest
```

Comment:
According to `python/setup.py`, the complete list should also include `scipy` and `psutil`, but not `pytest`.

Reply:
I was following the instructions literally on a new ubuntu 20.04 install and these are the changes I needed to make to get the example to work.

Reply:
I see. Which example did you try? In general if users only want to use instead of developing TVM, we should not ask them to install unnecessary packages such as `pytest` and `mypy`.

IMHO, we could indicate an example at the end of this doc to make it clear. It might be better to also add descriptions about installing extra requirements for developments (e.g., testing).

Reply:
Perhaps we can add a new section
* If you want to run unit tests
    pip3 install --user ...

Reply:
That's a good idea to me 👍

Reply:
I reverted this change

Reply:
Also, we should fix this properly with an official `requirements.txt` or `setup.py` set of dependencies, so people don't have to [roll their own lists](https://github.com/tlc-pack/tlcpack/blob/master/docker/install/centos_install_python_package.sh#L13-L16).

Reply:
@rkimball would you prefer to add the unit test section in this PR or file a follow-up PR?

Reply:
I think follow up PR to do what @binarybana suggests